scripts/upgrade

```diff
@@ -63,6 +63,19 @@ ynh_add_fpm_config --phpversion=$phpversion --usage=$fpm_usage --footprint=$fpm_
 # Create a dedicated NGINX config
 ynh_add_nginx_config
 
+#=================================================
+# ADD A CONFIGURATION
+#=================================================
+ynh_script_progression --message="Adding a configuration file..." --weight=1
+
+email=$(ynh_user_get_info --username=$admin --key=mail)
+
+# Setup application config
+ynh_add_config --template="../conf/.env.example" --destination="$install_dir/.env"
```

Comment:
I'm not sure of the purpose of this upgrade script, but if you replace the `.env` file of an installed app, some values generated during install, like the APP_KEY, will be lost. As APP_KEY is used by the encryption engine to encrypt the database,  doing so will make user data undecipherable.